pastas/modelcompare.py

```diff
@@ -0,0 +1,877 @@
+from itertools import combinations
+
+import matplotlib.pyplot as plt
+import numpy as np
+from pandas import DataFrame, concat
+
+import pastas as ps
+from warnings import warn
+
+
+class CompareModels:
```

Comment:
I like this new comparison class, but it's still a little confusing when you want to create your own comparison figure. Perhaps we can take a look for 1.0 to see if we can make this a bit more intuitive:
- Initializing your own axes mosaic
- Where do you pass the stressmodel dictionary, telling the comparison on which axes to model which contribution.
- Using .plot() for custom comparison figures?